### PR TITLE
[FIX] l10n_ph: cannot open vendor bill form view in generate 2307 wizard 

### PR DESCRIPTION
**Pull request by tong-odoo**
Steps to reproduce:
- Create a vendor bill and add an invoice line with taxes from l10n_ph
- Open generate 2307 wizard view
- Open the vendor bill form view from the wizard view

Current behavior:
Exception prompt.

Expected behavior:
Should be able to see the vendor bill inside the wizard view in readonly
mode.

Explanation:
With this modification there are two changes:

1) Wizard is using FormRenderer while account.move form is using
AccountMoveFormRenderer. Therefore it doesn't have a reference of the
default form view `AccountMoveFormNotebook`. To fix the issue, I copy the
view from account.move to make it work.

2) When no product is set on the invoice line, generating 2307 report will display
exception due to cannot find the product name. Now it should show empty when no
product is set on the invoice line.

X-original-commit: https://github.com/odoo-dev/odoo/commit/bd11387ea2c480c0266355f431305c276db0b628

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tong-odoo** [comment]
Hi @FlorianGilbert. it seems like there are some front-end breaks on v16. May I ask if you could help me to take a look at this PR?

File: addons/l10n_ph/wizard/generate_2307_wizard_views.xml
Comment (FlorianGilbert) [inline addons/l10n_ph/wizard/generate_2307_wizard_views.xml:62]:
```suggestion
                                <label for="invoice_date" string="Invoice Date" class="fw-bold" attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt'))]}"/>
```

File: addons/l10n_ph/wizard/generate_2307_wizard_views.xml
Comment (FlorianGilbert) [inline addons/l10n_ph/wizard/generate_2307_wizard_views.xml:63]:
```suggestion
                                <label for="invoice_date" string="Bill Date" class="fw-bold" attrs="{'invisible': [('move_type', 'not in', ('in_invoice', 'in_refund', 'in_receipt'))]}"/>
```

**Review by FlorianGilbert — CHANGES_REQUESTED** [review]
Hello, some little changes.

Probably need a functional check from @tsb-odoo

**Comment by tong-odoo** [comment]
> Hello, some little changes.
> 
> Probably need a functional check from @tsb-odoo

Updated

**Comment by FlorianGilbert** [comment]
Hello @tong-odoo 
It looks like you have to rebase your branch
image
Thank you :)

**Comment by tong-odoo** [comment]
@FlorianGilbert Updated thanks!

**Comment by tong-odoo** [comment]
@FlorianGilbert I re-based on the latest 16.0 branch. Please feel free to let me know if anything more needed from me.

**Comment by FlorianGilbert** [comment]
> @FlorianGilbert I re-based on the latest 16.0 branch. Please feel free to let me know if anything more needed from me.

I'm waiting for functional review from @tsb-odoo

**Comment by fja-odoo** [comment]
@tsb-odoo @FlorianGilbert Friendly reminder :)

**Comment by fja-odoo** [comment]
@FlorianGilbert @tsb-odoo Hello, please don't forget this fix

**Comment by tsb-odoo** [comment]
Hi, 
The PR looks obsolete to us. The wizard opens without any problem as far as I can see. 
Though, we noticed a traceback when no product is set on the invoice line if I understant correctly. 
Can you please update the PR (either close it or give additional info)
+ solve the issue. 
See tranceback : https://pastebin.com/QkiTiRFa
@fja-odoo @tong-odoo

**Comment by tong-odoo** [comment]
Hi @tsb-odoo, thanks for taking a look!

While the wizard view open problem is fixed, the form view inside the wizard still exist. I update the commit to fix the empty product invoice line error as well. Please take a look.

**Pull request closed without merging**
